docs/preview/features/message-pumps/customization.md

```diff
@@ -72,4 +72,40 @@ public void ConfigureServices(IServiceCollection services)
 > Note that the order in which the message handlers are registered is important in the message processing.
 > In the example, when a message handler above this one is registered that could also handle the message (same message type) than that handler may be chosen instead of the one with the specific filter.
 
+## Fallback message handling
+
+When receiving a message on the message pump and none of the registered `IMessageHandler`'s can correctly process the message, the message pump normally throws and logs an exception.
+It could also happen in a scenario that's to be expected that some received messages will not be processed correctly (or you don't want them to).
```

Comment:
New line for doc rendering
```suggestion

It could also happen in a scenario that's to be expected that some received messages will not be processed correctly (or you don't want them to).
```